qiskit_experiments/quantum_volume/qv_analysis.py

```diff
@@ -0,0 +1,282 @@
+
+        # calculate the experimental heavy output probability
+        return heavy_output_counts / circ_shots
+
+    @staticmethod
+    def _calc_z_value(mean, sigma):
+        """Calculate z value using mean and sigma.
+
+        Args:
+            mean (float): mean
+            sigma (float): standard deviation
+
+        Returns:
+            float: z_value in standard normal distibution.
+        """
+
+        if sigma == 0:
+            # assign a small value for sigma if sigma = 0
+            sigma = 1e-10
+            warnings.warn("Standard deviation sigma should not be zero.")
+
+        z_value = (mean - 2 / 3) / sigma
+
+        return z_value
+
+    @staticmethod
+    def _calc_confidence_level(z_value):
+        """Calculate confidence level using z value.
+
+        Accumulative probability for standard normal distribution
+        in [-z, +infinity] is 1/2 (1 + erf(z/sqrt(2))),
+        where z = (X - mu)/sigma = (hmean - 2/3)/sigma
+
+        Args:
+            z_value (float): z value in in standard normal distibution.
+
+        Returns:
+            float: confidence level in decimal (not percentage).
+        """
+
+        confidence_level = 0.5 * (1 + math.erf(z_value / 2 ** 0.5))
+
+        return confidence_level
+
+    def _calc_quantum_volume(self, heavy_output_prob_exp, depth, trials):
+        """
+        calc the quantum volume of the analysed system.
+        quantum volume is determined by the largest successful depth.
+        A depth is successful if it has 'mean heavy-output probability' > 2/3 with confidence
+        level > 0.977 (corresponding to z_value = 2).
+        we assume the error (standard deviation) of the heavy output probability is due to a
+        binomial distribution. standard deviation for binomial distribution is sqrt(np(1-p)),
+        where n is the number of trials and p is the success probability.
+
+        Returns:
+            dict: quantum volume calculations -
+            the quantum volume,
+            whether the results passed the threshold,
+            the confidence fof the result,
+            the heavy output probability for each trial,
+            the mean heavy output probability,
+            the error of the heavy output probability,
+            the depth of the circuit,
+            the number of trials ran
+        """
+        quantum_volume = 0
+        success = False
+        confidence_level_threshold = self._calc_confidence_level(z_value=2)
+        mean_hop = np.mean(heavy_output_prob_exp)
+        sigma_hop = mean_hop * ((1.0 - mean_hop) / trials) ** 0.5
+
+        z_value = self._calc_z_value(mean_hop, sigma_hop)
+        confidence_level = self._calc_confidence_level(z_value)
+        if mean_hop > 2 / 3 and confidence_level > confidence_level_threshold:
+            quantum_volume = 2 ** depth
+            success = True
+
+        result = {
+            "quantum volume": quantum_volume,
+            "success": success,
```

Comment:
Hi, I am using your quantum volume code for my project and I noticed that the "success" key is overwritten by the similar key in the analysis result: https://github.com/Qiskit/qiskit-experiments/blob/be3e21e461ef1a30babea6b4f1e4a718a25abbac/qiskit_experiments/base_analysis.py#L94.
It's confusing and also when quantum_volume = 0, it means that the quantum volume fails the test(not the experiment). So maybe you should change that name to another.

Reply:
thank you for the comment. "success" was change into "qv success". in addition, quantum volume for a run which failed to pass the treshold changed into 1